src/visualization/visualize_crmp.py

```diff
@@ -128,7 +128,7 @@ def net_production_estimators_and_time_steps():
     x = np.arange(len(x_labels))
     width = 0.15
     bar_labels = [
-        'NetCRM, mse', 'Linear Regression, mse', 'Bayesian Ridge, mse',
```

Comment:
Put these plots into a different file, rename all the files to just be the model name, and verify that they actually work. Perhaps in reverse order to what I just stated.